File: app/src/main/java/com/loskon/noteminimalism3/backup/BackupFilter.java
package com.loskon.noteminimalism3.backup;

import java.io.File;

/**
 * Фильтр для получения файлов с расширением .db
 */

public class BackupFilter {
    public static File[] getListFile(File folder) {
        return folder.listFiles((dir, name) -> name.toLowerCase().endsWith(".db"));
    }
}

File: app/src/main/java/com/loskon/noteminimalism3/backup/BackupFilter.kt
package com.loskon.noteminimalism3.backup

import java.io.File

/**
 * Фильтр для файлов с расширением .db
 */

class BackupFilter {

    companion object {

        fun getListDateBaseFile(folder: File?): Array<File>? {
            return folder?.listFiles { _, name: String ->
                name.lowercase().endsWith(".db")
            }
        }
    }
}